tests/testthat/test-predict_race.R

```diff
@@ -7,13 +7,10 @@ options("wru_data_wd" = TRUE)
 test_that("Tests surname only predictions", {
   set.seed(42)
   data(voters)
-  census <- readRDS(test_path("data/census_test_nj_block_2010.rds"))
```

Comment:
Removing this makes it necesary to have a census key, and that's not currently available in CI. Should leave the default census.data in.

Reply:
No. `surname.only` predictions should not require census data at all.